python/cudf/cudf/utils/ioutils.py

```diff
@@ -559,6 +559,10 @@
     size in bytes. Set the size to zero to read all data after the offset
     location. Reads the row that starts before or at the end of the range,
     even if it ends after the end of the range.
+keep_quotes : bool, default False
+    This parameter is only supported in ``cudf_experimental`` engine.
+    If `True`, the quotes characters of string values are preserved.
+    If `False`, the quotes characters of string values are not preserved.
```

Comment:
After reading this description, I was not really sure what `keep_quotes` does. I thought it meant that by default cudf does "quote normalisation" (e.g. always converts `'` to `"` as the delimiters), but that's a source-level thing so that can't be it.

What this really means is "when reading strings, treat the start and end delimiters as a part of the string". So perhaps:

> If `True`, any string values are read literally (and wrapped in an additional set of quotes). If `False` string values are parsed into Python strings.

Maybe add a simple example too?

WDYT?

Reply:
Done